tests/beaker_tests/vni/test_vni.rb

```diff
@@ -0,0 +1,197 @@
+###############################################################################
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+# TestCase Name:
+# -------------
+# test-vni.rb
+#
+# TestCase Prerequisites:
+# -----------------------
+# This is a Puppet vni resource testcase for Puppet Agent on
+# Nexus devices.
+# The test case assumes the following prerequisites are already satisfied:
+#   - Host configuration file contains agent and master information.
+#   - SSH is enabled on the N9K Agent.
+#   - Puppet master/server is started.
+#   - Puppet agent certificate has been signed on the Puppet master/server.
+#
+# TestCase:
+# ---------
+# This vni resource test verifies default values for all properties.
+#
+# The following exit_codes are validated for Puppet, Vegas shell and
+# Bash shell commands.
+#
+# Vegas and Bash Shell Commands:
+# 0   - successful command execution
+# > 0 - failed command execution.
+#
+# Puppet Commands:
+# 0 - no changes have occurred
+# 1 - errors have occurred,
+# 2 - changes have occurred
+# 4 - failures have occurred and
+# 6 - changes and failures have occurred.
+#
+# NOTE: 0 is the default exit_code checked in Beaker::DSL::Helpers::on() method.
+#
+# The test cases use RegExp pattern matching on stdout or output IO
+# instance attributes to verify resource properties.
+#
+###############################################################################
+
+require File.expand_path('../../lib/utilitylib.rb', __FILE__)
+
+# -----------------------------
+# Common settings and variables
+# -----------------------------
+testheader = 'Resource cisco_vni'
+
+# Define PUPPETMASTER_MANIFESTPATH.
+UtilityLib.set_manifest_path(master, self)
```

Comment:
You don't need this `UtilityLib.set_manifest` now that https://github.com/cisco/cisco-network-puppet-module/pull/108 is merged